<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesPage</class>
 <widget class="QWidget" name="PreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首选项</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   
   <!-- 标题 -->
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>⚙️ 首选项</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px; font-weight: bold; color: #2c3e50;</string>
     </property>
    </widget>
   </item>
   
   <!-- 外观设置 -->
   <item>
    <widget class="QGroupBox" name="appearanceGroup">
     <property name="title">
      <string>外观</string>
     </property>
     <layout class="QFormLayout" name="appearanceLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="themeLabel">
        <property name="text">
         <string>主题:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="themeCombo">
        <item>
         <property name="text">
          <string>浅色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>深色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>跟随系统</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>语言:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="languageCombo">
        <item>
         <property name="text">
          <string>简体中文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fontSizeLabel">
        <property name="text">
         <string>字体大小:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="fontSizeSpin">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
        <property name="suffix">
         <string> pt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <!-- 行为设置 -->
   <item>
    <widget class="QGroupBox" name="behaviorGroup">
     <property name="title">
      <string>行为</string>
     </property>
     <layout class="QFormLayout" name="behaviorLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="autoSaveCheck">
        <property name="text">
         <string>启用自动保存</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="autoSaveIntervalLabel">
        <property name="text">
         <string>自动保存间隔:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="autoSaveIntervalSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="suffix">
         <string> 分钟</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="minimizeToTrayCheck">
        <property name="text">
         <string>最小化到系统托盘</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="startupCheck">
        <property name="text">
         <string>开机自动启动</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
   <!-- 占位符 -->
   <item>
    <spacer name="mainSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   
   <!-- 状态栏 -->
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>就绪</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 5px;</string>
     </property>
    </widget>
   </item>
   
   <!-- 按钮栏 -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="restoreDefaultsButton">
       <property name="text">
        <string>恢复默认</string>
       </property>
       <property name="toolTip">
        <string>恢复所有设置为默认值</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>重置</string>
       </property>
       <property name="toolTip">
        <string>放弃更改，恢复上次保存的设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>应用</string>
       </property>
       <property name="toolTip">
        <string>保存并应用所有设置</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background-color: #27ae60; 
    color: white; 
    padding: 8px 24px;
    font-weight: bold;
}
QPushButton:hover { background-color: #229954; }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
